test/unit/filter_test.rb

```diff
@@ -90,6 +90,14 @@ class FilterTest < ActiveSupport::TestCase
       end
     end
 
+    test 'filter is not automatically scoped to to taxonomies' do
```

Comment:
Typo in the test description, `scoped to any` taxonomies.

In case you want to save the variable assignment (original_org) below:

```
@organization, Organization.current = Organization.current, @organization
assert_empty Filter.new.organizations
Organization.current, @organization = @organization, Organization.current
```